crates/bevy_pbr/src/lib.rs

```diff
@@ -217,6 +220,10 @@ impl Plugin for PbrPlugin {
                 RenderStage::Extract,
                 render::extract_clusters.label(RenderLightSystems::ExtractClusters),
             )
+            .add_system_to_stage(
+                RenderStage::Extract,
+                render::extract_ambient_light.label(RenderLightSystems::ExtractAmbientLights),
+            )
```

Comment:
If the extraction is just a simple for-loop that copies the AmbientLight as-is consider using `ExtractComponentPlugin` (but tbf, this might be a bit more verbose than just defining the extract system and not worth it)